flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartitionTest.java

```diff
@@ -250,40 +300,6 @@ public void testReleaseWhileWriting() throws Exception {
         }
     }
 
-    @Test
-    public void testReleaseWhileReading() throws Exception {
```

Comment:
I think this case can be changed a bit to a `testRelease` to verify that readers are released and files are deleted when actively releasing a `SortMergeResultPartition`.